cookbook/deployment/heroku.rst

```diff
@@ -305,20 +305,20 @@ This is also very useful to build assets on the production system, e.g. with Ass
 
     With the next deploy, Heroku compiles your app using the Node.js buildpack and
     your npm packages become installed. On the other hand, your ``composer.json`` is
-    now ignored. To compile your app with both buildpacks, Node.js *and* PHP, you can
-    use a special `multiple buildpack`_. To override buildpack auto-detection, you
+    now ignored. To compile your app with both buildpacks, Node.js *and* PHP, you need 
+    use both buildpacks. To override buildpack auto-detection, you
     need to explicitly set the buildpack URL:
 
     .. code-block:: bash
 
-        $ heroku buildpacks:set https://github.com/ddollar/heroku-buildpack-multi.git
-
-    Next, add a ``.buildpacks`` file to your project, listing the buildpacks you need:
-
-    .. code-block:: text
-
-        https://github.com/heroku/heroku-buildpack-nodejs.git
-        https://github.com/heroku/heroku-buildpack-php.git
+        $ heroku buildpacks:set heroku/nodejs
+        Buildpack set. Next release on your-application will use heroku/nodejs.
+        Run git push heroku master to create a new release using this buildpack.
+        $ heroku buildpacks:set heroku/php --index 2
```

Comment:
It sort of is, but the problem is that build pack order matters. In order to be able to use a composer post-install hook for e.g. Gulp, the node build pack needs to come before the PHP build pack. Just using `buildpacks:add` makes the process depend on the state of the application. The process I describe is idempotent.